test/metabase/models/collection_test.clj

```diff
@@ -1215,13 +1215,6 @@
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
 (deftest personal-collections-restrictions-test
-  (testing "Make sure we're not allowed to *unarchive* a Personal Collection"
```

Comment:
rather than deleting, maybe we should revert this test to test we could archive Personal collections?